tools/vscode/package.json

```diff
@@ -12,11 +12,12 @@
   "author": "Astro",
   "license": "MIT",
   "publisher": "astro-build",
+  "private": true,
```

Comment:
Can't release because only Nate has haccess.

@natemoo-re I'm not sure who created the `NPM_TOKEN` that we have in GitHub, but if you can give that person release permissions in npm then we can remove this.

https://www.npmjs.com/package/astro-vscode

otherwise, this just to make sure that the package isn't released for now

Reply:
Oops! I know I tried to give y'all tokens but was hitting some 500 errors on VS Code's end. Definitely worth trying again, hopefully they've fixed it.